ui/raidboss/data/03-hw/raid/a2s.js

```diff
@@ -90,18 +90,10 @@
     {
       'locale': 'fr',
       'replaceSync': {
-        'Added new combatant Boomtype Magitek Gobwalker G-VII': 'Ajout d\'un nouveau combattant Gobblindé magitek G-VII Boumbardier',
-        'Added new combatant Giant Bomb': 'Ajout d\'un nouveau combattant Bombe géante',
-        'Added new combatant Gordian Hardhelm': 'Ajout d\'un nouveau combattant Casque-dur gordien',
-        'Added new combatant Gordian Hardmind': 'Ajout d\'un nouveau combattant Cerveau-dur gordien',
-        'Added new combatant Gordian Sniper G-IX': 'Ajout d\'un nouveau combattant Sniper gordien',
-        'Added new combatant Jagd Doll': 'Ajout d\'un nouveau combattant Poupée jagd',
```

Comment:
And just for future reference, you don't need the "Added new combatant" text.  That is emitted from the ffxiv act plugin in English no matter what language your game is in.  It's sufficient to just say `'Jagd Doll': 'Poupée jagd',` and that will cover everything.